src/lib.rs

```diff
@@ -441,7 +445,7 @@ impl<T> OptionExt<T> for Option<T> {
 /// ```
 pub trait ErrorCompat {
```

Comment:
Should we even bother exposing an `ErrorCompat` when this feature is enabled?

Keeping it enabled seems like it would allow the smoothest ride to future rust.

Reply:
I think so, yes. For as long as we want to support backtraces on stable Rust before the standard library stabilizes, we'll need to have something. If we were to remove it, then the compatibility shim wouldn't be providing much value to those users.